includes/wikia/services/tests/SassInlineImageFilterTest.php

```diff
@@ -1,18 +1,18 @@
 <?php
 
-class SassBase64FilterTest extends WikiaBaseTest
+class SassInlineImageFilterTest extends WikiaBaseTest
 {
 	/**
 	 * @dataProvider getProcessData
 	 */
 	public function testProcess($input, $output) {
-		$filter = $this->getMockBuilder( 'Wikia\Sass\Filter\Base64Filter' )
+		$filter = $this->getMockBuilder( 'Wikia\Sass\Filter\InlineImageFilter' )
 			->disableOriginalConstructor()
-			->setMethods( [ 'encodeFile' ] )
+			->setMethods( [ 'inlineFile' ] )
 			->getMock();
 
 		$filter
-			->method( 'encodeFile' )
+			->method( 'inlineFile' )
```

Comment:
:bow: for base64 annotations that my grep didn't caught

I'll add additional unit test which coven new functionality.